coralproject/talk
# Dutch translation

## What does this PR do?

Translated all text fragments into Dutch.

## How do I test this PR?

Find someone who speaks Dutch, configure and run. 

Kidding aside, I think it will be implicitly tested by us (and perhaps others) moving forward and start deploying talk configured using the Dutch language. Then small details that need tweaking will show themselves.

----
We did reviews within the team and tried to have labeling of things as consistent as possible.
No brand-specific labels are in this translation, so every Dutch site should be able to use it.

I would expect minor updates later on, as we haven't seen all label in use yet. (Things like text being too long for the space available.)
